libr/core/cmd_anal.c

```diff
@@ -6915,18 +6919,23 @@ static void cmd_anal_aav(RCore *core, const char *input) {
 			//TODO: Reduce multiple hits for same addr
 			from = r_itv_begin (map2->itv);
 			to = r_itv_end (map2->itv);
-			eprintf ("Value from 0x%08"PFMT64x " to 0x%08" PFMT64x "\n", from, to);
+			snprintf (out, 256, "Value from 0x%08"PFMT64x " to 0x%08" PFMT64x " (aav)", from, to);
```

Comment:
dont hardcode buffer sizes. use sdb_fmt(), r_str_newf or go for sizeof(out)

Reply:
done